src/services/mpc/config.ts

```diff
@@ -0,0 +1,45 @@
+enum SocialWalletOptionsKeys {
+  web3AuthClientId = 'web3AuthClientId',
+  web3AuthAggregateVerifierId = 'web3AuthAggregateVerifierId',
+  web3AuthSubverifierId = 'web3AuthSubverifierId',
+  googleClientId = 'googleClientId',
+}
+
+export type SocialWalletOptions = {
+  [SocialWalletOptionsKeys.web3AuthClientId]: string
+  [SocialWalletOptionsKeys.web3AuthAggregateVerifierId]: string
+  [SocialWalletOptionsKeys.web3AuthSubverifierId]: string
+  [SocialWalletOptionsKeys.googleClientId]: string
+}
+
+export const isSocialWalletOptions = (options: unknown): options is SocialWalletOptions => {
+  if (typeof options !== 'object' || options === null) {
+    return false
+  }
+  const keys = Object.keys(options)
+
+  const hasRequiredKeys =
+    keys.length === 4 &&
+    keys.includes(SocialWalletOptionsKeys.web3AuthClientId) &&
```

Comment:
This validation is a bit excessive/boilerplatey

Reply:
Changed it to use the `in` operator to check if all keys exist in the object.